developer_manual/app_publishing_maintenance/upgrade-guide.rst

```diff
@@ -79,6 +87,11 @@ The original `PSR-0` standard was deprecated in 2014 and therefore the support f
 .. _`PSR-0`: https://www.php-fig.org/psr/psr-0/
 .. _`PSR-4`: https://www.php-fig.org/psr/psr-4/
 
+Last version with database.xml support and migration
+****************************************************
+
+Nextcloud 21 is the last major release that supports an app's ``appinfo/database.xml`` to define the database schema. This is your last chance to automatically convert this deprecated file into the new migration classes using ``occ migrations:generate-from-schema``.
```

Comment:
note: I brought this back from a previous PR but removed the links because the pages/sections have been removed as well, for the occ command.